Automattic/wp-calypso
# Hosting Command Palette: Add a few account commands

Closes https://github.com/Automattic/dotcom-forge/issues/4829

## Proposed Changes

This PR adds three new account commands:

* `Change plan on a site` - this command navigates to `/plans/{yourSite}`
* `Manage plan on a site` - this command navigates to `/plans/my-plan/{yourSite}`

<img width="813" alt="Screenshot 2023-12-11 at 4 33 44 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/87adc05c-2e8d-4ed0-bbf1-c640de5b5ab0">

For these two commands, I have set the credit card as an icon but am happy to change it to something else. In my opinion, I am not sure if we need both commands - I feel like `Change plan on a site` could be useful as it brings you to the overview of all the plans and this is where we can encourage the customer to upgrade. However, `/plans/my-plan/` (`Manage my plan`) page does not seem to be too useful for me - I don't know how often I would navigate to that page offer to see the overview of all the features that come with my plan. 

* `Get help` - this command opens a modal for the help center where the customers can search for questions

<img width="1279" alt="Screenshot 2023-12-11 at 4 32 50 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/bfb79ce8-53e1-4673-9813-324162d173aa">

The position of the modal is what it is on other pages - not sure if we need to adjust it a bit more aligned. If so, we might need to discuss what would be a better position.

## Testing Instructions

* Navigate to Calypso Live Link
* Navigate to `sites` page
* Press `cmd + k` or `ctrl + k` to trigger the command palette
* Search for `Change plan on a site` command
* Select a site to change plan
* Confirm you are being brought to  `/plans/{yourSite}`
---
* Navigate to `sites` page
* Press `cmd + k` or `ctrl + k` to trigger the command palette
* Search for `Manage plan on a site` command
* Select a site to change plan
* Confirm you are being brought to  `/plans/my-plan/{yourSite}`
---
* Navigate to `sites` page
* Press `cmd + k` or `ctrl + k` to trigger the command palette
* Search for `Get help`
* Click on the command and confirm that the help modal appears


## Pre-merge Checklist
